TeamCode/src/main/java/org/firstinspires/ftc/teamcode/TeleNew.java

```diff
@@ -38,134 +38,87 @@ public void init() {
-        }
-        if (gamepad1.left_stick_y < 0.3) {
-            wheel1.setPower(0.7);
-            wheel2.setPower(-0.7);
-            wheel3.setPower(0.7);
-            wheel4.setPower(-0.7);
-        }
-        if (gamepad1.left_stick_y > -0.3) {
-            wheel1.setPower(-0.7);
-            wheel2.setPower(0.7);
-            wheel3.setPower(-0.7);
-            wheel4.setPower(0.7);
-        }
-        if (gamepad1.left_stick_x < 0.3) {
-            wheel1.setPower(0.7);
-            wheel2.setPower(0.7);
-            wheel3.setPower(-0.7);
-            wheel4.setPower(-0.7);
-        }
-        if (gamepad1.left_stick_x > -0.3) {
-            wheel1.setPower(-0.7);
-            wheel2.setPower(-0.7);
-            wheel3.setPower(0.7);
-            wheel4.setPower(0.7);
-        }
+            //Neutral condition for horizontal
+        if (gamepad1.left_stick_x > -0.3 && gamepad1.left_stick_x < 0.3) { wheelMove (0, 0, 0, 0); }
 
-        if (gamepad1.left_bumper) {
-            wheel1.setPower(0.6);
-            wheel2.setPower(0.6);
-            wheel3.setPower(0.6);
-            wheel4.setPower(0.6);
-        }
-        if (gamepad1.right_bumper) {
-            wheel1.setPower(-0.6);
-            wheel2.setPower(-0.6);
-            wheel3.setPower(-0.6);
-            wheel4.setPower(-0.6);
-        }
+            //Go up
+        if (gamepad1.left_stick_y < 0.3) { wheelMove (0.7, -0.7, 0.7, -0.7); }
 
-       if (gamepad1.right_stick_y > -0.3 && gamepad1.right_stick_y < 0.3) {
-            wheel1.setPower(0);
-            wheel2.setPower(0);
-            wheel3.setPower(0);
-            wheel4.setPower(0);
-        }
-        if (gamepad1.right_stick_x > -0.3 && gamepad1.left_stick_x < 0.3) {
-            wheel1.setPower(0);
-            wheel2.setPower(0);
-            wheel3.setPower(0);
-            wheel4.setPower(0);
-        }
-        if (gamepad1.right_stick_y < 0.3) {
-            wheel1.setPower(0.35);
-            wheel2.setPower(-0.35);
-            wheel3.setPower(0.35);
-            wheel4.setPower(-0.35);
-        }
-        if (gamepad1.right_stick_y > -0.3) {
-            wheel1.setPower(-0.35);
-            wheel2.setPower(0.35);
-            wheel3.setPower(-0.35);
-            wheel4.setPower(0.35);
-        }
-        if (gamepad1.right_stick_x < 0.3) {
-            wheel1.setPower(0.35);
-            wheel2.setPower(0.35);
-            wheel3.setPower(-0.35);
-            wheel4.setPower(-0.35);
-        }
-        if (gamepad1.right_stick_x > -0.3) {
-            wheel1.setPower(-0.35);
-            wheel2.setPower(-0.35);
-            wheel3.setPower(0.35);
-            wheel4.setPower(0.35);
-        }
+            //Go down
+        if (gamepad1.left_stick_y > -0.3) { wheelMove (-0.7, 0.7, -0.7, 0.7); }
 
-        if (gamepad1.left_trigger == 1) {
-            claw.setPosition(openClaw);
-        }
+            //Go right
+        if (gamepad1.left_stick_x < 0.3) { wheelMove (0.7, 0.7, -0.7, -0.7); }
```

Comment:
What happens if `gamepad1.left_stick_y < 0.3` and `gamepad1.left_stick_x < 0.3`? Don't both assign a `wheelMove`?

Reply:
The code will execute both functions at the same time which may cause a problem which we need to fix

Reply:
Are you sure it won't simply just do the second one immediately and ignore the first one? I'm not sure what happens if you issue multiple wheel commands simultaneously, though that is basically happening within this `while` loop.

Reply:
pretty sure this can be written as {wheelMove (0.7,-0.7);}

Reply:
I'm not entirely sure but I think it would just execute the second button but we will need to test this and see what happens

Reply:
@curlyhair952 You need to look more closely at the [wheelMove](https://github.com/suchirchikkava/FTC-2022-2023-Season/pull/5/files#diff-e2d64a6ba57b191eaee42191396cd784fff73977abd6b26a78f0e5d01294802bR116-R121) function.

Reply:
Same here for the comment below, both should execute at once it looks like it is nearly instantaneous and looks like the two are going at the same time but its going one after another in a very short amount of time.